dev/tasks/build.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* jshint node: true */
+
+'use strict';
+
+module.exports = function( grunt ) {
+	grunt.registerTask( 'build', 'Build a release out of the current development code.', function() {
+		var done = this.async();
+		module.exports.build( done );
+	} );
+};
+
+// Exports the build method so it can be used from plain node code as well.
+module.exports.build = function( done ) {
```

Comment:
@mlewand 
> Can't we create a class for the builder?

Good suggestion. I've just done that with c3fc22c.

> And then wouldn't it be good to have unit tests for builder too?

I thought about that but I didn't want to spend time on this as there is no infrastructure for such tests in place. There are too many things to be bootstrapped right now, so I would propose to move ahead without tests here and make this a future enhancement.